### ISSUE TITLE
Node Status broken on v1.9.4

### ISSUE THREAD

**Issue by Ifiht** [rocket-pool/smartnode#369]
Using Rocketpool v1.9.4 newly updated (both binary and docker stack), I'm getting the below output for `node status`:
```
Your Smartnode is currently using the Ethereum Mainnet.

Could not get node status: Could not encode API response: json: unsupported value: +Inf
```
Current ETH1 = Besu, ETH2 = Lighthouse

Any suggestions?

**Comment by jclapis** [comment]
Hey, thanks for the report - this has been fixed and will be released in v1.9.5 - just waiting for Lighthouse to push their own update before I launch it. Should be tomorrow-ish.

**Comment by Ifiht** [comment]
For what it's worth - can confirm good fix. Thanks!!
